core/src/components/toast/toast.tsx

```diff
@@ -77,7 +78,7 @@ export class Toast implements ComponentInterface, OverlayInterface {
    * How many milliseconds to wait before hiding the toast. By default, it will show
    * until `dismiss()` is called.
    */
-  @Prop() duration = 0;
+  @Prop() duration = config.get('toastDuration', 0);
```

Comment:
Can we use `config.getNumber`? This method has typing to return `number` which should also remove the `component.d.ts` diff.